### ISSUE TITLE
Gunicorn is getting 404 errors

### ISSUE THREAD

**Issue by avishayil** [markstory/lint-review#218]
Hello,
I'm running lint-review on docker. I noticed that:

```
web_1     | [2018-08-07 08:44:59 +0000] [10] [DEBUG] GET /ping
web_1     | 127.0.0.1 - - [07/Aug/2018:08:44:59 +0000] "GET /ping HTTP/1.1" 404 233 "-" "curl/7.47.0"
broker_1  | 2018-08-07 08:45:01.810 [info] <0.813.0> accepting AMQP connection <0.813.0> (192.168.128.3:38632 -> 192.168.128.2:5672)
broker_1  | 2018-08-07 08:45:01.815 [info] <0.813.0> connection <0.813.0> (192.168.128.3:38632 -> 192.168.128.2:5672): user 'guest' authenticated and granted access to vhost '/'
broker_1  | 2018-08-07 08:45:01.861 [warning] <0.813.0> closing AMQP connection <0.813.0> (192.168.128.3:38632 -> 192.168.128.2:5672, vhost: '/', user: 'guest'):
```

Gunicorn is getting 404 errors on ping.
Any ideas?

**Comment by markstory** [comment]
A similar error was reported in #109

**Comment by avishayil** [comment]
If it matters, it happens on Amazon Ubuntu EC2

**Comment by markstory** [comment]
I'll see if I can reproduce locally. From the various issue reports around docker it sounds like something is not working well out of the box.

**Comment by victorvianna** [comment]
Exactly the same error here!

**Comment by avishayil** [comment]
To elaborate more, I had to manually use the Dockerfile instead of your
docker image and add the curl library to the web container.

**Comment by victorvianna** [comment]
You mean the healthcheck is failing just because curl is not installed in the container?

**Comment by avishayil** [comment]
Correct. Apart from that, I encounter the 404 error

**Comment by avishayil** [comment]
@markstory after doing exactly what you did:
````
{
  "Status": "unhealthy",
  "FailingStreak": 13,
  "Log": [
    {
      "Start": "2018-08-09T06:38:46.846977419Z",
      "End": "2018-08-09T06:38:46.930840768Z",
      "ExitCode": 127,
      "Output": "/code/docker_healthcheck.sh: line 5: curl: command not found\n"
    },
    {
      "Start": "2018-08-09T06:38:56.938449019Z",
      "End": "2018-08-09T06:38:57.024881763Z",
      "ExitCode": 127,
      "Output": "/code/docker_healthcheck.sh: line 5: curl: command not found\n"
    },
    {
      "Start": "2018-08-09T06:39:07.031830471Z",
      "End": "2018-08-09T06:39:07.108661863Z",
      "ExitCode": 127,
      "Output": "/code/docker_healthcheck.sh: line 5: curl: command not found\n"
    },
    {
      "Start": "2018-08-09T06:39:17.115100898Z",
      "End": "2018-08-09T06:39:17.200192834Z",
      "ExitCode": 127,
      "Output": "/code/docker_healthcheck.sh: line 5: curl: command not found\n"
    },
    {
      "Start": "2018-08-09T06:39:27.206339846Z",
      "End": "2018-08-09T06:39:27.284374255Z",
      "ExitCode": 127,
      "Output": "/code/docker_healthcheck.sh: line 5: curl: command not found\n"
    }
  ]
}
````

**Comment by avishayil** [comment]
When i'm trying to access ip:5000
````
Not Found
The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
````

**Comment by avishayil** [comment]
after installing curl on the container:
````
{
  "Status": "unhealthy",
  "FailingStreak": 13,
  "Log": [
    {
      "Start": "2018-08-09T07:03:50.251230455Z",
      "End": "2018-08-09T07:03:50.335706263Z",
      "ExitCode": 127,
      "Output": "/code/docker_healthcheck.sh: line 5: curl: command not found\n"
    },
    {
      "Start": "2018-08-09T07:04:00.342142965Z",
      "End": "2018-08-09T07:04:00.42743846Z",
      "ExitCode": 127,
      "Output": "/code/docker_healthcheck.sh: line 5: curl: command not found\n"
    },
    {
      "Start": "2018-08-09T07:04:10.435744311Z",
      "End": "2018-08-09T07:04:10.514360236Z",
      "ExitCode": 127,
      "Output": "/code/docker_healthcheck.sh: line 5: curl: command not found\n"
    },
    {
      "Start": "2018-08-09T07:04:20.521903589Z",
      "End": "2018-08-09T07:04:20.632672584Z",
      "ExitCode": 127,
      "Output": "/code/docker_healthcheck.sh: line 5: curl: command not found\n"
    },
    {
      "Start": "2018-08-09T07:04:30.641181574Z",
      "End": "2018-08-09T07:04:30.729361121Z",
      "ExitCode": 22,
      "Output": "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n                                 Dload  Upload   Total   Spent    Left  Speed\n\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\ncurl: (22) The requested URL returned error: 404 NOT FOUND\n"
    }
  ]
}
````

**Comment by avishayil** [comment]
Environment:
AWS EC2 - Ubuntu Server 16.04
Installed docker-ce, docker-compose with the instructions of official docker documentation, then cloning the repository, running docker-compose pull and docker-compose up.

**Comment by markstory** [comment]
Ok. `curl` does seem to be missing from the application container. I can fix that. 

What is the sha of the docker image you get from `docker-compose pull`? You can get that from `docker images | grep 'lint-review'`
